app/mailers/notify_mailer.rb

```diff
@@ -5,4 +5,13 @@ def email_template(user, template)
     set_template(template)
     mail(to: user.email)
   end
+
+  def idea_email_template(user, idea, template)
```

Comment:
Is it a good idea to make this reusable? I feel like these will be quite specific depending on the email template being used, it might be better to make them explicitly for the template we are using in each case. What do you think?